sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
Wait ... don't we also want to tie this to `is_funded`? Otherwise we reset *all* dues back to 0, even for folks with a good card attached.